src/api/app/components/notification_action_description_component.rb

```diff
@@ -34,9 +35,20 @@ def call
           concat(tag.p("User '#{@notification.notifiable.user.login}' created a report for a #{@notification.event_payload['reportable_type']} for the following reason:"))
           concat(tag.p(@notification.event_payload['reason']))
         end
+      when 'Event::ClearedDecision'
+        capture do
+          concat(tag.p("User '#{@notification.notifiable.moderator.login}' cleared a report for #{@notification.notifiable.reports.first.reportable} for the following reason:"))
```

Comment:
I'd omit the word `User` (here and in the other similar cases) and leave just the login value in the sentence:
`Iggy cleared a report........`

even better if it could be the either in bold or the link to the user page.